script/world.js

```diff
@@ -49,7 +49,7 @@ var World = {
 			cooldown: 2
 		},
 		'bone spear': {
-			verb: _('stab'),
+			verb: _('throw'),
```

Comment:
How about 'lunge'

Reply:
Yeah. that'll work. I didn't know that many verbs for it, so I just used that one.